snorkel/learning/gen_learning.py

```diff
@@ -377,10 +412,20 @@ def _compile(self, L, y, init_acc, init_deps, init_class_prior):
             w_off = 0
 
         for i in range(w_off, w_off + n):
-            weight[i]['isFixed'] = False
-            weight[i]['initialValue'] = np.float64(init_acc)
+            # Prior on LF acc
+            if (LF_priors[i] != 0.5):
+                weight[w_off]['isFixed'] = True
+                weight[w_off]['initialValue'] = np.float64(0.5 * np.log(LF_priors[i] / (1 - LF_priors[i])))
+                w_off += 1
+            # Learnable acc for LF
+            if (not is_fixed[i]):
+                weight[w_off]['isFixed'] = False
+                if LF_priors[i] == 0.5:
+                    weight[w_off]['initialValue'] = np.float64(init_acc + .1 - .2 * self.rng.random())
+                else:
+                    weight[w_off]['initialValue'] = np.float64(0)
```

Comment:
@bryan-he Should we just have all LF accs initialize at the prior value, rather than having two separate ways of handling?  Thoughts here?